### PR TITLE
Add support for autoincrement columns

### PR DESCRIPTION
**Pull request by lafriks**
Both `ID` and `BigID` will now map to INTEGER as SQLite3 supports only INTEGER auto increment columns

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,10 +1,12 @@
 module github.com/go-rel/sqlite3
 
-go 1.20
+go 1.21
```
Comment (lafriks) [inline go.mod:3]: With installed go 1.21 this was the only way I could get to go mod tidy to work :/

Comment (lafriks) [reply go.mod:3]: oh, that's because go-rel has go 1.21 as min version set

File: sqlite3.go [hunk 2]
```diff
@@ -96,32 +96,55 @@ func errorMapper(err error) error {
...
 	return typ, m, n
 }
 
+func columnOptionsMapper(column *rel.Column) string {
+	var buffer strings.Builder
+
+	if column.Primary {
+		buffer.WriteString(" PRIMARY KEY")
+		if column.Type == rel.ID || column.Type == rel.BigID {
+			buffer.WriteString(" AUTOINCREMENT")
```
Comment (Fs02) [inline sqlite3.go:128]: can we add integration test for this?
I don't remember if we already have one
https://github.com/go-rel/sqlite3/blob/main/sqlite3_test.go#L25

Comment (lafriks) [reply sqlite3.go:128]: it should have as in spec there is ID/BigID columns

Comment (lafriks) [reply sqlite3.go:128]: https://github.com/go-rel/sql/pull/56

Comment (lafriks) [reply sqlite3.go:128]: updated

File: go.mod [hunk 3]
```diff
@@ -18,3 +20,5 @@ require (
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/go-rel/sql v0.15.1-0.20230926233117-4af81443c5e1 => ../sql/
```
Comment (Fs02) [inline go.mod:24]: is this intentional or was forgotten to remove?

Comment (lafriks) [reply go.mod:24]: It was forgotten to remove in one of previous PR

Comment (lafriks) [reply go.mod]: @Fs02 removed

**Review by Fs02 — APPROVED** [review]
LGTM, Thanks

**Comment by lafriks** [comment]
@Fs02 can you please merge this?

**Comment by Fs02** [comment]
oh sorry, I thought I've merged this

**Pull request merged**